components/Tabs/src/MDCTabBar.h

```diff
@@ -33,9 +33,13 @@
  @see https://material.io/guidelines/components/tabs.html
  */
 IB_DESIGNABLE
-@interface MDCTabBar : UIView
+@interface MDCTabBar : UIView <UIBarPositioning>
 
-/** Return the desired height for the tab bar given an item appearance. */
+/** The default height for the tab bar given a position and item appearance. */
++ (CGFloat)defaultHeightForPosition:(UIBarPosition)position
+                     itemAppearance:(MDCTabBarItemAppearance)appearance;
+
+/** The default height for the tab bar in the top position, given an item appearance. */
 + (CGFloat)defaultHeightForItemAppearance:(MDCTabBarItemAppearance)appearance;
```

Comment:
Can we make this an instance method and use self.barPosition?
I'm a big fan of class methods when we don't need state, but in this case it seems like we need it.